functions.php

```diff
@@ -50,7 +54,7 @@ function create_event_tax()
         )
     );
 }
-
+*/
```

Comment:
There were warnings being output to the debug log because of this piece of code,

Reply:
I think this went back to when we were experimenting with taxonomies? good find.